samples/detect.v1p1beta1.js

```diff
@@ -452,68 +120,43 @@ function detectWeb(fileName) {
     .then(results => {
       const webDetection = results[0].webDetection;
 
-      if (webDetection.fullMatchingImages.length) {
-        console.log(
-          `Full matches found: ${webDetection.fullMatchingImages.length}`
-        );
-        webDetection.fullMatchingImages.forEach(image => {
-          console.log(`  URL: ${image.url}`);
-          console.log(`  Score: ${image.score}`);
-        });
-      }
-
-      if (webDetection.partialMatchingImages.length) {
-        console.log(
-          `Partial matches found: ${webDetection.partialMatchingImages.length}`
-        );
-        webDetection.partialMatchingImages.forEach(image => {
-          console.log(`  URL: ${image.url}`);
-          console.log(`  Score: ${image.score}`);
-        });
-      }
-
-      if (webDetection.webEntities.length) {
-        console.log(`Web entities found: ${webDetection.webEntities.length}`);
-        webDetection.webEntities.forEach(webEntity => {
-          console.log(`  Description: ${webEntity.description}`);
-          console.log(`  Score: ${webEntity.score}`);
+      if (webDetection.bestGuessLabels.length) {
```

Comment:
Is `webDetection.bestGuessLabels` guaranteed to be an array (and not `null` or `undefined`)?
If so, this `if` block is unnecessary (but hurts nothing). If not, then the `if` is insufficient because an exception will be raised saying "length is not a property of undefined".